apps/backend/.gitignore

```diff
@@ -6,4 +6,6 @@
 
 # misc
 .env
-test.env
\ No newline at end of file
+test.env
+.vscode
+.gitignore
```

Comment:
why are we ignoring .gitignore?

Reply:
I have seen many people do that. They dont even want to reveal what they are hiding, lol.